bcda/auth/plugin/alpha.go

```diff
@@ -0,0 +1,43 @@
+package auth
+
+import (
+	"errors"
+)
+
+type AlphaAuthPlugin struct{}
+
+func (p *AlphaAuthPlugin) RegisterClient(params []byte) error {
```

Comment:
RegisterClient(params []byte) ([]byte, error)

Need a return value; it will contain (among other things) the id by which the AuthProvider knows this client. Otherwise, we don't know what id to tell it to perform all the client-related methods on.